src/GitVersionCore.Tests/BuildServers/GitHubActionsTests.cs

```diff
@@ -22,13 +22,13 @@ public void SetUp()
             log = new NullLog();
 
             environment = new TestEnvironment();
-            environment.SetEnvironmentVariable("GITHUB_ACTION", Guid.NewGuid().ToString());
+            environment.SetEnvironmentVariable("GITHUB_ACTIONS", "true");
```

Comment:
do you mind changing with constant from GitHubActions.EnvironmentVariableName

Reply:
sure